src/components/EditImageModal/EditImageModal.scss

```diff
@@ -1,74 +1,87 @@
 @import 'edx-bootstrap';
 
-.SFE-wrapper {
```

Comment:
I'm undoing this nesting here, since we figured out how to avoid it, hence why a lot of this file has been modified because of indentation.

Reply:
If you add `?w=1` to the end of the URL it won't show any of the whitespace changes :)